code/renderers/vue3/src/public-types.ts

```diff
@@ -27,11 +34,35 @@ export type StoryFn<TArgs = Args> = AnnotatedStoryFn<VueFramework, TArgs>;
  *
  * @see [Named Story exports](https://storybook.js.org/docs/formats/component-story-format/#named-story-exports)
  */
-export type StoryObj<TArgs = Args> = StoryAnnotations<VueFramework, TArgs>;
+export type StoryObj<MetaOrCmpOrArgs = Args> = MetaOrCmpOrArgs extends {
```

Comment:
```suggestion
export type StoryObj<TMetaOrCmpOrArgs = Args> = MetaOrCmpOrArgs extends {
```

Generally we've gone with a convention that a type variable is prefaced with `T` to distinguish from a concrete type. WDYT?

Reply:
I don't know, not a big fan, but also not necessarily against it. I understand that it may be something useful if you want to do:

```ts
type GenericType<TComponent extends Component> = ...
```

But I'm also not sure if it really becomes more readable if you use it everywhere.

I'm okay with adopting it, I would have to change some stuff then in the other renderers as well :D

Reply:
Shall I adopt this convention in a separate PR?